tests/state/test_state_transition.py

```diff
@@ -125,3 +132,45 @@ def test_recent_attesters_added(genesis_crystallized_state,
             (base + (active_state.height - 1) % epoch_length) * attester_count
 
         parent_block = child_block
+
+
+def test_recent_attester_processing(genesis_crystallized_state, config, attester_reward):
+    zero_deltas = process_recent_attesters(genesis_crystallized_state, [], config=config)
+    assert len(zero_deltas) == genesis_crystallized_state.num_active_validators
+    assert all(delta == 0 for delta in zero_deltas)
+
+    some_validators = [0, 5, 39]
+    some_deltas = process_recent_attesters(
+        genesis_crystallized_state,
+        some_validators,
+        config=config,
+    )
+    assert len(some_deltas) == genesis_crystallized_state.num_active_validators
+    assert all(
+        delta == attester_reward
+        for i, delta in enumerate(some_deltas)
+        if i in some_validators
+    )
```

Comment:
Can `assert all([... for ...])` be translated into
```
for ...
    assert ...
```
so that test error would not be buried to the list?